### ISSUE TITLE
Can't run project

### ISSUE THREAD

**Issue by dwarcher** [deprecatedcoder/PictureWindow#1]
Hi! Thanks for putting this on github - I was looking to try something similar. I'm having trouble getting this working - seems like there are a number of missing scripts and nothing is happening after I follow the readme. Another question - do I need to position the Picture window component relative to my monitor position? A little explanation of the PictureWindow and CompanionUI components would be appreciated.

**Comment by deprecatedcoder** [comment]
No problem, sorry to hear you're having issues with it.

First I should ask what version of Unity you are using.  This was built on 5.5.2, and I haven't tested anything earlier, so that's one thing to consider.

If that's up to date, could you elaborate on the errors you are having?

You don't need to place anything relative to your monitor, all PictureWindow and CompanionUI need is to be present in the scene.

PictureWindow manages the setup and use of the window.  CompanionUI is just the on-screen instructions.

**Comment by dwarcher** [comment]
Ok, I upgraded to 5.5 and started over and prefabs that were broken before are working now. One thing I noticed - it seems like your calculations may assume a specific orientation for your chaperone / play space. The demo target space was flipped so I was looking at mostly skybox. Once I rotated my SteamVR camera rig by 180 then everything seemed correct. Also, there were still a few missing scripts on the target box prefab but I don't think they were important.
